testool/README.md

```diff
@@ -72,9 +72,9 @@ In the config file you define `[[suite]]`s that defines how tests will be execut
 - `max_steps` the maximum number of executed opcodes. If this is reached, the test is marked to be ignored.
 - `max_gas` the maximum gas of a test. If the is reached, the test is marked to be ignored.
 - `unimplemented_opcodes`, if any of these opcodes are found in the execution trace, the test is marked to be ignored.
```

Comment:
@adria0 should we remove this documentation about the deleted `unimplemented_opcodes`? or it has not been deleted?